core/schema/sdk.go

```diff
@@ -629,6 +629,31 @@ func (sdk *goSDK) baseWithCodegen(
 				},
 			},
 		},
+	}
+
+	// inject sdk specific env variables before withExec
+	// TODO(rajatjindal): do we want to have a safelist here?
+	// e.g. disallow system env variables?
```

Comment:
Potentially I would filter to anything with a `GO` prefix? Is there any reason we'd want to inject non-go prefixes?

We could always expand the list later, but I'm struggling to think of a legit reason we'd want anything that's not in that list.